app/components/primer/time_ago_component.ts

```diff
@@ -0,0 +1 @@
+import '@github/time-elements'
```

Comment:
shouldn't we only import the time-ago element? `import {TimeAgoElement} from '@github/time-elements'`

Reply:
We should but:
 
 - Our linter complains
 - Or compiler doesn't actually tree shake this as the whole file has side effects meaning no dead code elimination can take place

This is a piece of tech debt we should address upstream and patch in a follow up PR I think.